### ISSUE TITLE
1.19.4 Crash with meteor + sodium + iris shader.

### ISSUE THREAD

**Issue by HyCore** [MeteorDevelopment/meteor-client#3530]
### Describe the bug

Meteor fail to inject itself in sodium with shader:
`Critical injection failure: Redirector setColor(IF)I in meteor-client-sodium.mixins.json:SodiumBlockRendererMixin from mod meteor-client failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap meteor-client-refmap.json`

https://pastebin.mozilla.org/31Ha3d9A

### Steps to reproduce

```bash
Install latest iris and sodium version with latest meteor 1.19.4, and it will crash.
```

### Meteor Version

0.5.3

### Minecraft Version

1.19.4

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by EvanFarwolder** [comment]
Same issue.

**Comment by Mathug53fa** [comment]
I have the same problem pls patch :'c

**Comment by PeakXing** [comment]
yes，Same issue.

**Comment by MP3Martin** [comment]
`meteor client v0.5.3 build #1815` works for me, `meteor client v0.5.3 build #1816` crashes

**Comment by HyCore** [comment]
> [`meteor client v0.5.3 build #1815`](https://meteorclient.com/download?devBuild=1815) works for me, [`meteor client v0.5.3 build #1816`](https://meteorclient.com/download?devBuild=1816) crashes

what version of sodium and iris where you using ?

**Comment by sussyGaymer** [comment]
> [`meteor client v0.5.3 build #1815`](https://meteorclient.com/download?devBuild=1815) works for me, [`meteor client v0.5.3 build #1816`](https://meteorclient.com/download?devBuild=1816) crashes

Had the issue, this fixed it.

**Comment by HyCore** [comment]
not on the latest i think

**Comment by bartaback** [comment]
Same issue for me with latest iris and sodium for 1.19.4 with meteor build 1816 and later

**Comment by bartaback** [comment]
Same issue for me with latest iris and sodium for 1.19.4 with meteor build 1816 and later

**Comment by Funtimes909** [comment]
Bumping this, I also have the same issue

**Comment by LuukPlay** [comment]
Same issue.

**Comment by pO7HncXR** [comment]
Same issue.

**Comment by sussyGaymer** [comment]
please restrict comments to contributors, i don't want to get an email anytime someone comments "same issue here"

**Comment by HyCore** [comment]
> please restrict comments to contributors, i don't want to get an email anytime someone comments "same issue here"

Just unsubscribe to it. The only notification/email you will get and if it get closed one day or if you get mentioned.

**Comment by LuukPlay** [comment]
meteor-client-0.5.3-1821 + iris-mc1.19.4-1.6.1
still with problem

**Comment by Enstrayed** [comment]
> [`meteor client v0.5.3 build #1815`](https://meteorclient.com/download?devBuild=1815) works for me, [`meteor client v0.5.3 build #1816`](https://meteorclient.com/download?devBuild=1816) and newer crashes

Can confirm that using build 1815 instead works. With meteor enabled, loading any world will cause a crash with:
`java.lang.RuntimeException: Mixin transformation of me.jellysquid.mods.sodium.client.render.chunk.compile.pipeline.BlockRenderer failed`
This does not occur with Meteor enabled, suggesting it is at fault.

**Comment by h4rmy** [comment]
sodium has nothing to do with it, its the iris whats causing the crash

**Comment by AstraxVZ** [comment]
for me, it doesn't crash but instead most of the features don't work

**Comment by Tanminghao25** [comment]
I hope this issue can be fixed on 1.21 before.
But the developers of Meteor did not.
This issue still exists, which leads to the lack of Continuity.

**Comment by Tanminghao25** [comment]
> sodium has nothing to do with it, its the iris whats causing the crash钠与此无关，是虹膜导致了崩溃

In my error report, it shows that the crash was caused by the incompatibility between Sodium and Meteor.
And it will also collapse without iris
